beacon_node/src/cli.rs

```diff
@@ -452,6 +451,16 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .requires("execution-endpoint")
                 .takes_value(true)
         )
+        .arg(
+            Arg::with_name("execution-jwt-secret-key")
+                .long("execution-jwt-secret-key")
+                .value_name("EXECUTION-JWT-SECRET-KEY")
+                .alias("jwt-secret-key")
+                .help("Hex-encoded JWT secret for the \
+                       execution endpoint provided in the --execution-endpoint flag.")
+                .requires("execution-endpoint")
```

Comment:
It might be nice to also add a `.conflicts_with("execution-jwt")` here to prevent the user from providing both values and wondering why one doesn't work.

Reply:
good catch, I've fixed it